src/RscChart.tsx

```diff
@@ -104,14 +106,24 @@ export const RscChart = forwardRef<ChartHandle, RscChartProps>(
 
 		const sanitizedChildren = sanitizeRscChartChildren(props.children);
 
-		// THE MAGIC, builds our spec
+		const [rerender, setRerender] = useState<boolean>(true);
+
+		const invertRerender = useRef(() => setRerender(!rerender));
+
+		useEffect(() => {
+			doAnimateFromZero.current = true;
+			// ensure a rerender so the current chart animates.
+			invertRerender.current();
```

Comment:
.This is pretty funky!  I wouldn't expect that setting state in a ref would be different from just setting state directly in the useEffect.  But if those code does the job, it might be OK

Reply:
You were right - I think the ref was an artifact from a previous fix I was trying! It works (and looks) much better as a state variable, too.